Sources/SwiftSyntaxBuilder/BooleanLiteralExprConvenienceInitializers.swift

```diff
@@ -10,9 +10,11 @@
 //
 //===----------------------------------------------------------------------===//
 
+import SwiftSyntax
+
 extension BooleanLiteralExpr {
   public init(_ value: Bool) {
-    self.init(booleanLiteral: value ? .true : .false)
+    self.init(booleanLiteral: value ? TokenSyntax.true : TokenSyntax.false)
```

Comment:
This is now needed as we use `ExpressibleAsTokenSyntax` and not  `TokenSyntax`

Reply:
What do you think about changing `Tokens.swift` to extend `ExpressibleAsTokenSyntax` instead of `TokenSyntax`. IIUC we could still use leading dot syntax in that case.

Reply:
We get and `Contextual member reference to static property 'true' requires 'Self' constraint in the protocol extension`.

I'm not that familiar with static properties and protocols. 
I have tried to make it return `Self` instead of `ExpressibleAsTokenSyntax `. I also tried to return `TokenSyntax,  but no luck. `

Reply:
I honestly don’t fully understand *why* it works, but 
```swift
extension ExpressibleAsTokenSyntax where Self == TokenSyntax {
  static var `associatedtype`: TokenSyntax { SyntaxFactory.makeAssociatedtypeKeyword()
    .withTrailingTrivia(.spaces(1)) }
}
```
appears to work.

Reply:
No problem! 

Should we in the readme or another file make a Known issues list? 
Just so we remember it at some points and inform users of the library.